R/install-github.r

```diff
@@ -98,6 +106,10 @@ remote_download.github_remote <- function(x, quiet = FALSE) {
   } else {
     auth <- NULL
   }
+  ## if repo uses submodules warn user
```

Comment:
1. I think the comment is extraneous (given how simple the code is)
2. Can you please use a space after `if`
3. `warning()` looks like it's indented too much
4. I always use `call. = FALSE` with warning